### PR TITLE
Rename Ruby related classes/files to RuboCop

### PR DESCRIPTION
**Pull request by gylaz**
As part of the move to name files after the linters.

### CODE REVIEW

**Comment by gylaz** [comment]
[Upgrade Golint](https://trello.com/c/BK1ej5ux/822-upgrade-golint)

File: spec/services/resolve_config_aliases_spec.rb [hunk 1]
```diff
@@ -6,11 +6,12 @@
       it "renames them to the appropriate linter" do
         config = {
           "javascript" => { "enabled" => false },
-          "ruby" => { "enabled" => false },
+          "ruby" => { "enabled" => true },
+          "flog" => { "enabled" => false },
```
Comment (salbertson) [inline spec/services/resolve_config_aliases_spec.rb]: Why is `flog` included in the test?

Comment (gylaz) [reply spec/services/resolve_config_aliases_spec.rb]: Just an arbitrary key I used in the test.

**Comment by gylaz** [comment]
@salbertson I think `linters` will just pick it up off that queue.

**Pull request merged**